### PR TITLE
Blog app: data model

### PR DESCRIPTION
**Pull request by Adams-Ghub**
The following were carried out

- Set up project and configure linters
- Create migration files for all entities in the ER diagram
- Run migration files to generate schema and database

### CODE REVIEW

File: db/migrate/20230720125622_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Creates the posts table in the database
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :LikesCounter
+      t.string :CommentsCounter
```
Comment (danmainah) [inline db/migrate/20230720125622_create_posts.rb]: - [ ] @Adams-Ghub Great work on your migration file, however, this field should be of type integer and not of type string.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Adams-Ghub , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Setup the PostgreSQL database correctly.
✔️ Added all the tables and columns required.
✔️ Professional written README file with all sections required.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It would have been nice if you made the user_id column in comments and likes tables to author_id, this will make it in line with the post table.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: db/migrate/20230720125622_create_posts.rb (hunk 1 above)
Comment (rloterh) [reply db/migrate/20230720125622_create_posts.rb]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Adams,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on introducing an improvement to data type of in active model!

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**